src/components/ha-base-time-input.ts

```diff
@@ -400,6 +400,10 @@ export class HaBaseTimeInput extends LitElement {
       padding-inline-start: 4px;
       padding-inline-end: initial;
     }
+    ha-input-helper-text {
```

Comment:
Just move it out of the `.time-input-wrap-wrap` div?

Reply:
Yes I read it too late sorry :)